README.md

```diff
@@ -1,157 +1,74 @@
-      "type": "Mage"
-  }
-}
 ```
-
-```javascript
-/**
- * The place where all the magic happens.  This is the primary processing method for your bot which will always be called once per tick.
- * @param playerId - the numeric identifier for your bot within the tickInfo data.  This normally maps to the transform id for your bot when it represents a playable avatar
- * @param tickInfo - the json representing the tick info for your game at the current tick
- * @param matchInfo - DEPRECATED - To be removed
- * @param actionQueue - The queue to which you can append new actions to be performed based on your bot's processing of this tick.
- */
-runTurn(playerId, tickInfo, matchInfo, actionQueue) {
-  // Implement tick processing here
-  // Refer to the sample bots for examples of accessing state from the tickInfo or adding actions to the actionQueue
-}
+https://github.com/Regression-Games/RGUnityBots.git#0.0.14
 ```
 
-#### Sample Bots
-- Sample bot for Unity BossRoom that picks a random character and cycles to a new ability action on each tick... [AbilityBot](https://github.com/Regression-Games/UnityTestBot/blob/main/abilitybot/index.js)
-- Sample bot for Unity BossRoom that follows another player around and supports them... [CoopBot](https://github.com/Regression-Games/UnityTestBot/blob/main/coopbot/index.js)
-- Sample bot for Unity BossRoom that clicks menu buttons to start the game without human intervention... [MenuBot](https://github.com/Regression-Games/UnityTestBot/blob/main/menubot/index.js)
+Once you add the package, visit our [documentation site](https://docs.regression.gg) to get started! Also, visit https://play.regression.gg to create an account.
+
+## What is included in this package
+
+- Useful abstractions and classes for making bots within Unity
+- Easy integration approaches with [`RGState`](https://docs.regression.gg/studios/unity/unity-sdk/RGState) and [`RGAction`](https://docs.regression.gg/studios/unity/unity-sdk/RGAction) that allow bots to understand your game
+- An [`RGOverlayCanvas`](https://docs.regression.gg/studios/unity/tutorials/first_tutorial#add-the-rgoverlaycanvas) for in-game control of starting and stopping bots
+- An in-editor [timeline and replay feature](https://docs.regression.gg/studios/unity/unity-sdk/in-editor-replay) to see what your bots did during a run
+- Examples for [running bots in CI/CD environments](https://docs.regression.gg/studios/unity/tutorials/github_actions)
+- (Coming soon) Ready-to-go sample scenes for playing with existing bots
+
+## FAQ
+
+#### What languages do you support?
+
+We currently support C# for bots running locally within your Unity runtime, but
+also offer JavaScript and Typescript through our remote bot system. To make a 
+bot based on JavaScript and Typescript, visit the 
+[bot manager page](https://play.regression.gg/bots) to create a bot from a template.
+
+#### How can I see what my bot did?
+
+Every run with bots produces a replay file that can be downloaded from the 
```

Comment:
Not true 'yet' for C# bots.

Reply:
Will correct this!

Reply:
This looks to still need JS vs C# differentiation text